src/commands/PublishCommand.js

```diff
@@ -155,6 +155,34 @@ export default class PublishCommand extends Command {
   }
 
   getVersionsForUpdates(callback) {
+    if (this.flags.cdVersion) {
+      // Allows automatic bumping to next semver via cdVersion flag
+      if (this.flags.cdVersion === "patch" ||
+          this.flags.cdVersion === "minor" ||
+          this.flags.cdVersion === "major" ||
+          this.flags.cdVersion === "current"
+      ) {
+        // Warn about possible problems with git and current
```

Comment:
Kind of a long story having to do with our deployment pipeline but we came to the conclusion it was a better strategy to always version. Removing the option in latest changes